actioncable/lib/action_cable/channel/streams_handlers/base.rb

```diff
@@ -0,0 +1,23 @@
+module ActionCable
+  module Channel
+    module StreamsHandlers
+      class Base # :nodoc:
+        attr_reader :channel, :identifier
```

Comment:
I might be missing something, but is there are any reason why `channel` and `identifier` need to be part of the public API? 😬